Dynatrace/dynatrace-operator
# Fixes K8s OLM manifests to include serviceAccount for oneAgents

## Description

As we have removed the OpenshiftorOLM check in helm manifests, we have overseen the fact that we still need it for K8s OLM integration. This shows with the fact that the oneAgent Clusterrole was missing there.

Fixes #1940 

## How can this be tested?
Run and check if the `dynakube-oneagent` serviceAccount is included in the CSV => 

`export OLM=true && export PLATFORM="kubernetes" && make bundle VERSION="0.13.0" IMAGE="docker.io/dynatrace/dynatrace-operator" TAG="v0.13.0" OLM_IMAGE="docker.io/dynatrace/dynatrace-operator:v0.13.0"`

## Checklist
